src/base/QXmppMixConfigItem.h

```diff
@@ -0,0 +1,131 @@
+// SPDX-FileCopyrightText: 2023 Melvin Keskin <melvo@olomono.de>
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+#pragma once
```

Comment:
As I already [commented](https://github.com/qxmpp-project/qxmpp/pull/588#issuecomment-1817827616), QXmpp uses it. But I pushed the old usage for now because there is no resolution yet.